applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig $(dvfs_state)_ENABLE
+	bool "DVFS module $(dvfs_state)"
+	default y
+	depends on $(dvfs_dependencies)
+
+if $(dvfs_state)_ENABLE
+
+config $(dvfs_state)_TIMEOUT_MS
+	int "timeout"
+	default 5000
+	range 0 65535
+	help
+	  Timeout in milliseconds specifying time after which DVFS module will go back to
+	  low frequency after entering $(dvfs_state).
+
+choice "$(dvfs_state)_ACTIVE_FREQ"
+	prompt "Default active frequency of $(dvfs_state)"
+	default $(dvfs_state)_FREQ_HIGH
+
+config $(dvfs_state)_FREQ_HIGH
+	bool "High frequency"
+
+config $(dvfs_state)_FREQ_MEDLOW
+	bool "Medium low frequency"
+
+config $(dvfs_state)_FREQ_LOW
```

Comment:
Implementation cannot handle requesting for low frequency. No reason to define it here.